src/lib/events/events.ts

```diff
@@ -44,86 +44,88 @@ export default [
         tag: Tag.Other,
         state: State.Past,
         type: Type.Internal,
-        event: 'ASCENS10N',
+        event: 'Ascent: The 10th Anniversary',
         dates: 'August 4 - August 6, 2023',
         description:
-            'On its tenth year, UP CSI celebrates a decade of innovation and excellence with its members and alumni who became a part of this incredible journey.',
+            "UP CSI’s 10th anniversary was celebrated with a series of fun-filled activities, including a game night, an open-tambay day, and, all highlighting the organization's growth and bond over the years.",
         src: ascenc10n,
         href: '/events/',
     },
     {
         tag: Tag.EX,
         state: State.Past,
         type: Type.External,
-        event: 'EX Series 2023',
+        event: 'EX Series',
         dates: 'August 4 - August 6, 2023',
         description:
-            'Aute incididunt officia exercitation ea aliqua officia culpa incididunt proident aliqua commodo velit.',
+            'EX Series is an initiative of UP CSI that provides company talks and workshops which give students a glimpse of what work is like once they graduate.',
         src: ex2023,
         href: '/events/',
     },
     {
         tag: Tag.App,
-        state: State.Past,
-        type: Type.Internal,
-        event: 'CSFlight',
+        state: State.Present,
+        type: Type.External,
+        event: 'Application Process',
         dates: 'August 4 - August 6, 2023',
         description:
-            'Qui eu voluptate non nostrud. Et irure reprehenderit pariatur nulla laboris ipsum ut tempor magna officia tempor mollit Lorem reprehenderit.',
-        src: csflight,
+            'The application process serves as the recruitment phase of UP CSI. During this process, applicants undergo necessary training and development to prepare them for membership in the organization.',
```

Comment:
I don't think most of these description texts don't fit in the cards, so it might be good to adjust either the card sizing or text sizing slightly.

You don't need to handle this since it's not really part of your task right now, we can put this up as a separate minor issue for whoever wants to contribute. Still, it should be mentioned as a problem.